config/consensus.go

```diff
@@ -940,25 +960,9 @@ func initConsensusProtocols() {
 	vFuture.CompactCertWeightThreshold = (1 << 32) * 30 / 100
 	vFuture.CompactCertSecKQ = 128
 
-	vFuture.EnableKeyregCoherencyCheck = true
-
-	// Enable support for larger app program size
-	vFuture.MaxExtraAppProgramPages = 3
-	vFuture.MaxAppProgramLen = 2048
-
-	// enable the InitialRewardsRateCalculation fix
-	vFuture.InitialRewardsRateCalculation = true
```

Comment:
the following `vFuture.PaysetCommit = PaysetCommitMerkle` can also be dropped.

Reply:
done